### DISCUSSION TITLE
1.2.5

### DISCUSSION THREAD

**Discussion by Carseason** [Carseason/routerdog#32] [Announcements]
1. 优化  影视墙 UI
2. 优化 下载中心 UI
3. 优化 文件管理 UI
4. 修复 读取 cgi 时缓存问题 导致无法刷新配置
5. 替换桌面背景图片为: https://unsplash.com/photos/purple-gradient-bexwsdM5BCw
6. 修复 Docker 资源分配时默认分配 内存和 CPU
7. 增加右上角 行为日志 窗口 （暂无功能）

**Comment by Fengerxuanxiao** [comment]
任务计划能否添加一个运行历史纪录

**Reply by Carseason** [reply]
https://github.com/Carseason/routerdog/discussions/34
